pkg/asset/manifests/template.go

```diff
@@ -2,6 +2,12 @@ package manifests
 
 import "github.com/openshift/installer/pkg/types/baremetal"
 
+// AlibabaCloudCredsSecretData holds encoded credentials and is used to generate cloud-creds secret
+type AlibabaCloudCredsSecretData struct {
+	Base64encodeAccessKeyID     string
+	Base64encodeSecretAccessKey string
+}
```

Comment:
see above comments, I don't think we should be doing this for manual credential mode